app/styles/_mobile-clients.less

```diff
@@ -1,6 +1,28 @@
-.mobile-clients-view .copy-to-clipboard {
-  margin-bottom: 25px;
-  pre {
-    max-height: 350px;
+
+.create-binding {
+  text-align: center;
+  .icon {
+    width: 64px;
+    height: 64px;
```

Comment:
Is it possible to remove this height and add equal top/bottom padding to `.note`?